### DISCUSSION TITLE
Limited Use Links

### DISCUSSION THREAD

**Discussion by mister-svinia** [minio/minio#18482] [Q&A] [answered]
Reading the documentation, I did not find any such capabilities. I think I missed it (my English is perfect), I decided to check the arguments for creating a link in a Python shell function - and I didn't find it either. Is it really not there, or am I just not seeing it?

If this is really not the case, then such an opportunity would be very cool. And it would be even better if it could be combined with the "AND"/"OR" action time parameter.

**Comment by supriyo-biswas** [comment, marked as answer]
(We should move this to discussions.)

https://docs.aws.amazon.com/AmazonS3/latest/userguide/ShareObjectPreSignedURL.html
https://min.io/docs/minio/linux/reference/minio-mc/mc-share-download.html
